README.md

```diff
@@ -0,0 +1,28 @@
+# GitHub Actions - Add PR to project
+
+Add community-created pull requests to the [Community Contributions project](https://github.com/orgs/silverstripe/projects/4) when they're opened.
+
+## Usage
+
+**.github/workflows/add-pr-to-project.yml**
+```yml
+name: Add pull request to project
+
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
Question for reviewer:

Do we want to skip draft PRs? If so I can add `ready_for_review` as one of the "types", and update the action script to exit early for draft PRs.

Reply:
yes skip draft PR's, they're not ready for review at that point

Reply:
Done

Reply:
Should we be doing this instead? https://stackoverflow.com/questions/68349031/only-run-actions-on-non-draft-pull-request - events like `reopened` we might want to also trigger this action?

Reply:
reopening PRs is handled by the built-in logic for projects.